src/components/navBar/menuList.js

```diff
@@ -0,0 +1,19 @@
+const menuList = [
+  {
+    title: 'Currencies',
+    url: '/',
+    id: 0,
+  },
+  {
+    title: 'Converter',
+    url: '/converter',
+    id: 1,
+  },
+  // {
+  //   title: 'Details',
+  //   url: '/currencies/:details',
+  //   id: 3,
+  // },
```

Comment:
- [ ] Kindly remove these commented codes. This will make your codebase cleaner and more professional